modules/cli/src/main/java/org/apache/ignite/internal/cli/call/connect/ConnectCall.java

```diff
@@ -61,7 +68,7 @@ public CallOutput<String> execute(ConnectCallInput input) {
             stateConfigProvider.get().setProperty(ConfigConstants.LAST_CONNECTED_URL, nodeUrl);
             session.setJdbcUrl(constructJdbcUrl(configuration, nodeUrl));
             session.setConnectedToNode(true);
-
```

Comment:
Please keep the line breaks intact, they are helpful for readability. There are some more missing line breaks in other files.